portfolio/src/main/java/com/google/sps/servlets/InteractiveChartServlet.java

```diff
@@ -10,27 +16,46 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** TODO(briafassler): store data to a database */
+/** Servlet for the interactive pie chart */
 @WebServlet("/interactive-chart")
 public class InteractiveChartServlet extends HttpServlet {
   private Map<String, Integer> emotionVotes = new HashMap<>();
-  private static String EMOTION_DATA_PARAM = "emotion";
+  private static String EMOTION_DATA = "emotion";
+  private static String VOTES_PARAM = "Votes";
+  private static String TIMESTAMP_PARAM = "timestamp";
 
```

Comment:
nit: for consistency with other constants i think VOTES and TIMESTAMP are good

Reply:
done